.github/scripts/buildActions.sh

```diff
@@ -28,12 +28,26 @@ declare -r NOTE_DONT_EDIT='/**
  */
 '
 
-for ACTION in "${GITHUB_ACTIONS[@]}"; do
-    # Build the action
-    ACTION_DIR=$(dirname "$ACTION")
-    ncc build "$ACTION" -o "$ACTION_DIR"
-
-    # Prepend the warning note to the top of the compiled file
-    OUTPUT_FILE="$ACTION_DIR/index.js"
-    echo "$NOTE_DONT_EDIT$(cat "$OUTPUT_FILE")" > "$OUTPUT_FILE"
+# This stores all the process IDs of the ncc commands so they can run in parallel
+declare ASYNC_BUILDS
+
+for ((i=0; i < ${#GITHUB_ACTIONS[@]}; i++)); do
```

Comment:
NAB: TIL that `${#GITHUB_ACTIONS[@]}` and `${#GITHUB_ACTIONS[*]}` will return the number of elements in the `GITHUB_ACTIONS` array. Very cool!